endpoints/events/accounts_test.go

```diff
@@ -0,0 +1,141 @@
+package events
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/stored_requests"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+// Mock AccountsFetcher
+type mockAccountsFetcher struct {
+	Fail          bool
+	Error         error
+	EventsEnabled bool
+}
+
+func (e *mockAccountsFetcher) FetchAccount(ctx context.Context, accountID string) (json.RawMessage, []error) {
+	if e.Fail {
+		return nil, []error{e.Error}
+	}
+
+	var acc = &config.Account{
+		ID:            accountID,
+		EventsEnabled: false,
+	}
+
+	if e.EventsEnabled {
+		acc.EventsEnabled = true
+	}
+
+	s, _ := json.Marshal(acc)
+
+	return s, []error{}
+}
+
+func TestShouldReturnDefaultAccountWithSpecifiedIdWhenAccountNotFound(t *testing.T) {
+	// prepare
```

Comment:
The structure of these tests all look very similar, which is a "code smell" that a [table drive test](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests) might make sense here. You can look in the privacy (among others) package for further examples

Reply:
These tests are not necessary if we use the existing GetAccount, which is covered by  https://github.com/prebid/prebid-server/blob/master/endpoints/openrtb2/auction_test.go#L1979-L2044

Reply:
These will likely be removed due to #1483

Reply:
Sounds good. We're reviewing that today.